tasks/pipeline.py

```diff
@@ -444,13 +444,15 @@ def generate_failure_messages(project_name: str, failed_jobs: FailedJobs) -> Dic
     return messages_to_send
 
 
-@task
-def trigger_child_pipeline(_, git_ref, project_name, variables="", follow=True):
+@task(iterable=['variable'])
+def trigger_child_pipeline(_, git_ref, project_name, variables="", follow=True, variable=None):
```

Comment:
Does it make sense to keep both `variables` and `variable`? I would suggest moving exclusively to the list-based approach to prevent confusion.

Reply:
I hesitated but was worried about the impact to existing code, but on a 2nd thought I think all callsites are changed in this PR :sweat_smile: I suppose you're right, I'll remove the old way